### DISCUSSION TITLE
Export Preferences to JSON/XML file?

### DISCUSSION THREAD

**Discussion by sameerpremji** [dbeaver/dbeaver#10602] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version: Windows 10 (x64)
- DBeaver version: 7.3.0

#### Connection specification:
- Database name and version: Not relevant to the question

**Comment by kseniiaguzeeva** [comment, marked as answer]
Please copy your workspace into another PC https://github.com/dbeaver/dbeaver/wiki/Workspace-Location.
